jupyterhub/templates/proxy/autohttps/rbac.yaml

```diff
@@ -83,9 +93,9 @@ roleRef:
   kind: ClusterRole
   name: nginx-{{ .Release.Name }}
 subjects:
-  - kind: ServiceAccount
-    name: autohttps
-    namespace: {{ .Release.Namespace }}
+- kind: ServiceAccount
```

Comment:
Regarding these indentation changes: there seem to be two standards: one where `-` is aligned with the parent element and another where `-` is indented like a dict. The latter seems more common and more readable (it's easier to see the levels this way), but this is converting to the former. Is there a reason to dedent this here (e.g. motivated by a helm linter)?

Reply:
#622 - The kubernetes/chart linting ignores the difference, allowing both. The repositories I've looked at have been mixing both. I've seen situations where both indentation variants have been the most readable. I'm happy to decide on any variant and go with it.

```yaml
parent1:
- key1: value1
- key2: value2

parent2:
  - key1: value1
  - key2: value2
```

I've never seen really ugly examples of the first kind of indentation, but some due to the second kind of indentation. But that is often not an issue.

Example of nested situation causing a big challenge no matter what kind of indentation scheme used, but I figure it would be easier to use the former kind as the key-names would always start two spaces in.

https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/c826f93e255992f2dd8e3c53d7ca2630b786ccc3/jupyterhub/values.yaml#L44-L49

I'm setting up some linting in #624 - we can configure it to however we please, I'm looking a directive to follow!